apm-agent-bootstrap/src/main/java/bootstrap/dispatcher/IndyBootstrapDispatcher.java

```diff
@@ -82,6 +83,18 @@ public static CallSite bootstrap(MethodHandles.Lookup lookup,
         return callSite;
     }
 
+    public static void logAdviceException(Throwable exception) {
+        try {
+            if (logAdviceException != null) {
+                logAdviceException.invoke(null, exception);
+            } else {
+                exception.printStackTrace();
+            }
+        } catch (Throwable t) {
+            t.printStackTrace();
```

Comment:
That's great, I missed the `Throwable#printStackTrace` usage in the exception handler we used so far when adding [`SystemStandardOutputLogger`](https://github.com/elastic/apm-agent-java/blob/main/apm-agent-common/src/main/java/co/elastic/apm/agent/common/util/SystemStandardOutputLogger.java#L58)!
Now we can avoid printing advice-based exceptions to std streams as well 🙂. 
You cannot use `SystemStandardOutputLogger` here, but please either add the same logic (look for the same system property or env variable that disables it, with consideration in security manager) or avoid using `printStackTrace`.
I am personally OK with not having a fallback at all as we expect our exception handler to always work.